packages/dd-trace/test/appsec/telemetry/rasp.spec.js

```diff
@@ -38,7 +38,7 @@ describe('Appsec Rasp Telemetry metrics', () => {
         appsecTelemetry.updateRaspRequestsMetricTags({
           duration: 42,
           durationExt: 52
-        }, req, 'rule-type')
+        }, req, { type: 'rule-type' })
```

Comment:
not your fault, but it's awful to see that these tests were wrong, but not failing :/